boards/arm/nucleo_wl55jc/nucleo_wl55jc.dts

```diff
@@ -101,6 +102,17 @@
 	status = "okay";
 };
 
+&subghzspi {
+	status = "okay";
+	lora: subghz-radio@0 {
```

Comment:
Node names should be generic. So in this case, it should be `radio@0` as we have a single radio at `0` index. By mistake we added node names based on device names like `sx1276`. I'll get them fixed.

Reply:
Cool will change that... how about the label property? Should that go to "RADIO" or something else?